lang/test/org/partiql/lang/eval/EvaluatingCompilerDateTimeTests.kt

```diff
@@ -34,4 +63,187 @@ class EvaluatingCompilerDateTimeTests : EvaluatorTestBase() {
             case("'2021-03-17' IS DATE", "false")
         )
     }
+
+    private fun createIonTimeStruct(value: Time): IonStruct =
+        ion.newEmptyStruct().apply {
+            add("hour", ion.newInt(value.hour))
+            add("minute", ion.newInt(value.minute))
+            add("second", ion.newFloat( value.second + value.nano * 10.0.pow(-9)))
+            add("timezone_hour", ion.newInt(value.tz_minutes?.div(60)))
+            add("timezone_minute", ion.newInt(value.tz_minutes?.rem(60)))
+            addTypeAnnotation("\$partiql_time")
+        }
```

Comment:
Is this literally mirroring the equivalent code in `ExprValue` somewhere?

Reply:
Yes. I might as well get rid of this assertion as it is mirroring the code.